packages/react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -1272,7 +1311,7 @@ class Dropdown extends AutoControlledComponent<WithAsProp<DropdownProps>, Dropdo
    * because we want to keep the event handling interface
    */
   setStateAndInvokeHandler = (
-    handlerName: keyof DropdownProps,
+    handlerNames: string[],
```

Comment:
Let's make this array of `keyof DropdownProps`.

Reply:
Actually `(keyof DropdownProps)[]`